Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -992,6 +993,12 @@ extension TelemetryWrapper {
             GleanMetrics.FirefoxHomePage.historyHighlightsGroupOpen.record()
         case (.action, .view, .historyImpressions, _, _):
             GleanMetrics.FirefoxHomePage.customizeHomepageButton.add()
+        case (.action, .view, .historyHighlightContextualMenu, _, let extras):
+            if let type = extras?[EventExtraKey.contextualMenuType.rawValue] as? String {
+                GleanMetrics.FirefoxHomePage.historyHighlightsContext.record(GleanMetrics.FirefoxHomePage.HistoryHighlightsContextExtra(type: type))
+            } else {
+                recordUninstrumentedMetrics(category: category, method: method, object: object, value: value, extras: extras)
+            }
```

Comment:
Unit tests for telemetry? 😸 Could be done on the Context menu helper tests in that case (which doesn't exists yet.. but would be great!) Or otherwise from the TelemetryWrapperTests